client/components/WithinCommunityByline/WithinCommunityByline.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import Byline, { BylineProps } from 'components/Byline/Byline';
+import { AttributionWithUser } from 'types';
+import { usePageContext } from 'utils/hooks';
+import { Tooltip, Icon } from '@blueprintjs/core';
+
+// Review Note; I bet there's a better way to do this?
+type OwnProps = Omit<BylineProps, 'contributors'> & {
+	contributors: AttributionWithUser[];
+};
+
+const WithinCommunityByline = (props: OwnProps) => {
+	const { communityData } = usePageContext();
+	const { contributors } = props;
+	let { bylinePrefix } = props;
+	// TODO: Add function to get the sort option from community settings, and add other sort options
+	if (communityData.id === '1a71ef4d-f6fe-40d3-8379-42fa2141db58') {
+		contributors.sort((last, next) => {
```

Comment:
Don't forget to avoid mutating this array! I'd suggest:

```ts
const sortedContributors = communityId === "..." ? sortContributorsByLastName(contributors) : contributors;
```
and passing that into `Byline`.